### PR TITLE
added a description to the readme file

### PR DESCRIPTION
**Pull request by Richez003**
created a portfolio using the ;
correct github flow
run linters to check for errors
crated a new branch

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="header">
+        <h2 class="h2">Welcome</h2>
```
Comment (VICTOR-CHUKWU) [inline index.html]: - [ ] According to the project requirement this logo should be inside `<a>` tag, this is for easy navigation when you are not in home page, kindly fix this

File: index.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+                 feature or website Look through some of my work and experience!
+                  If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.
+            </p>
+            <ul class="icon-list">
+                <li><img src="./images/twitter.png" alt="twitter"></li>
+                <li><img src="./images/Linkedin.png" alt="linkedin"></li>
+                <li><img src="./images/medium.png" alt="medium"></li>
+                <li><img src="./images/github.png" alt="github"></li>
+                <li><img src="./images/angellist.png" alt="angellist"></li>
```
Comment (VICTOR-CHUKWU) [inline index.html]: - [ ] Kindly put your images in an `a` tag so as to navigate to your social media when clicked. it should look like this ` <li><a href=''>twitter</a></li>`

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @Richez003 👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit
- All linters passed


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by moise10r — APPROVED** [review]
### Hi @Richez003 , 👋 

# STATUS: Approved  🎉 

Great work on making the changes requested by a previous reviewer :1st_place_medal:  
 Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 




## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @moise10r when you comment so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+            <ul class="icon-list">
+                <li><a href="https://twitter.com/richardfriday14"><img src="./images/twitter.png" alt="twitter"></a></li>
+                <li><a href="https://www.linkedin.com/in/richard-friday-54980718a"><img src="./images/Linkedin.png" alt="linkedin"></a></li>
+                <li><a><img src="./images/medium.png" alt="medium"></a></li>
+                <li><a href="https://github.com/Richez003"><img src="./images/github.png" alt="github"></a></li>
+                <li><a href="https://angel.co/jobs?ref=onboarding"><img src="./images/angellist.png" alt="angellist"></a></li>
+                </ul>
+                <span class="dis"> 
+                    <img src="./images/Disabled.png" alt="disabled">
+                </span>
```
Comment (moise10r) [inline index.html:39]: - [ OPTIONAL] Kindly make this down icon to be responsive for all mobile versions size :arrow_down: it must be always positioned to the centre, I think you can use `margin: 0 auto` to fix this


fool

**Pull request merged**